.jenkins/pytorch/test.sh

```diff
@@ -188,27 +188,22 @@ test_xla() {
   assert_git_not_dirty
 }
 
-if [ -z "${BUILD_ENVIRONMENT}" ] || [[ "${BUILD_ENVIRONMENT}" == *-test ]]; then
-  if [[ "${BUILD_ENVIRONMENT}" == *xla* ]]; then
-    test_torchvision
-    test_xla
-  else
-    test_torchvision
-    test_python_nn
-    test_python_all_except_nn
-    test_aten
-    test_libtorch
-    test_custom_script_ops
-  fi
+if [[ "${BUILD_ENVIRONMENT}" == *xla* ]]; then
+  test_torchvision
+  test_xla
+elif [[ "${BUILD_ENVIRONMENT}" == *-test1 ]]; then
+  test_torchvision
+  test_python_nn
+elif [[ "${BUILD_ENVIRONMENT}" == *-test2 ]]; then
+  test_python_all_except_nn
+  test_aten
+  test_libtorch
+  test_custom_script_ops
 else
-  if [[ "${BUILD_ENVIRONMENT}" == *-test1 ]]; then
-    test_torchvision
-    test_python_nn
-  elif [[ "${BUILD_ENVIRONMENT}" == *-test2 ]]; then
-    test_torchvision
```

Comment:
I think "test_torchvision" is duplicated here as test1 has already run it. @yf225

Reply:
Yes it's likely a duplicate, thanks for catching it :D

Reply:
Test in h

Reply:
Test in h